src/components/common/CompletingParty.vue

```diff
@@ -58,6 +60,18 @@ export default class CompletingParty extends Vue {
     return (this.validate && !this.completingPartyValid)
   }
 
+  // FUTURE: pull this from shared auth composable (after vue3 upgrade)
+  get isSbcStaff (): boolean {
+    const currentAccount = sessionStorage.getItem(SessionStorageKeys.CurrentAccount)
+    if (!currentAccount) return false
+    try {
+      return JSON.parse(currentAccount).accountType === AccountTypes.STAFF_SBC
+    } catch (error) {
+      console.log('Error parsing current account =', error)
+      return false
+    }
+  }
```

Comment:
Most of this code is already in App.vue

I think that all you need is a new getter like this in state-getters.ts:
```
/** Whether the current account is SBC Staff. */
export const isPremiumAccount = (state: StateIF): boolean => {
  return (state.stateModel.accountInformation.accountType === AccountTypes.SBC_STAFF)
}
```

Reply:
https://github.com/bcgov/business-edit-ui/blob/e1db511ab234c9b6a4abbd27f180c270d502c432/src/App.vue#L713

Reply:
ah nice okay will update